test/functional/test_framework/test_node.py

```diff
@@ -209,6 +217,44 @@ def is_node_stopped(self):
     def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
         wait_until(self.is_node_stopped, timeout=timeout)
 
+    def assert_start_raises_init_error(self, extra_args=None, expected_msg=None, match=ErrorMatch.FULL_TEXT, *args, **kwargs):
+        """Attempt to start the node and expect it to raise an error.
+
+        extra_args: extra arguments to pass through to pivxd
+        expected_msg: regex that stderr should match when pivxd fails
+
+        Will throw if pivxd starts without an error.
+        Will throw if an expected_msg is provided and it does not match pivxd's stdout."""
+        with tempfile.SpooledTemporaryFile(max_size=2**16) as log_stderr:
+            try:
+                self.start(extra_args, stderr=log_stderr, *args, **kwargs)
+                self.wait_for_rpc_connection()
+                self.stop_node()
+                self.wait_util_stopped()
```

Comment:
typo, should be `self.wait_until_stopped()`

Reply:
:+1: fixed

Reply:
Will be good to have the python-lint script catching this stuff.